### ISSUE TITLE
Tips or advices after last time using bots/gps fake location.

### ISSUE THREAD

**Issue by AlexCtasy** [finndev/PokeBuddy#1045]
Hello guys, Well tomorrow I was banned with that message of out of range, waited 12 hours and now is all fine, I boting my last lv to reach 30 and stop to boting. In my country a lot of people are 30, 33, 36 and of course boted people also, btw... What u guys advices to me..

After I get my last lv to 30 ill delete all bots...  Then about fake location i'm use that pokemon API for iphones, that u just install the pogo that is hacked on the phone by Cydia, not for jailbreak iphone.. Well I used it always and tomorrow I used in x1 or x2 speed and all was fine no ban visibles.. Btw.

Is better I delete that gps location and download the app official and stop with apps using? I guess niantic is just focused on ppl that are boting for now, not on ppl who use fake gps .. But well i wanna be safe..

So unnistall all sh1tty and keep online the original app one? Or stay with the fake gps location only.. What u say guys.

**Comment by TankardBR** [comment]
@AlexCtasy its possible to install fake location is IOS with no jailbreak? how?

**Comment by AlexCtasy** [comment]
@TankardBR Yes it's. Is like a pre-hacked pogo that u install with cydiaa impactor.

**Comment by TankardBR** [comment]
@AlexCtasy I searched the internet and found many thanks. Sry my english

**Comment by AlexCtasy** [comment]
@TankardBR Are u using it right now on ur iphone? Yes u cant teleport, is just to have a human touch in another places that u cant go.. Because where I lives, dont have any decent pokespot or a good area to catch pokemons, is an area that u just have ratata xoaoxoa.. So just run and x2.

**Comment by TankardBR** [comment]
@AlexCtasy same here Hahahahah, and this blue flag and house in app for what use?

**Comment by TankardBR** [comment]
@AlexCtasy its possible save some locations?

**Comment by AlexCtasy** [comment]
@TankardBR of course, i saved mine fake location, as forever! So when i use it from my phone, wherever i are, ill be saved on same cords.
